chroma_core/models/filesystem.py

```diff
@@ -109,6 +109,15 @@ def get_servers(self):
         # (http://stackoverflow.com/questions/4764110/django-template-cant-loop-defaultdict)
         return dict(servers)
 
+    def get_server_groups(self):
```

Comment:
Is this needed in this patch?

Reply:
Helpful "example"?  (I've been trying to get this function in for a while) I need it for hotpools

Reply:
Ah gotcha. Ok just wondering.